src/Tribe/Context/locations.php

```diff
@@ -76,6 +76,25 @@
 	],
 	'post_type' => [
 		'read' => [
+			Tribe__Context::FUNC        => static function() {
+				$post_type_objs = get_post_types( [
+					'public' => true,
+					'_builtin' => false,
+				], 'objects' );
+
+				foreach( $post_type_objs as $post_type ) {
+					if ( empty( $post_type->query_var ) ) {
+						continue;
+					}
+
+					$url_value = tribe_get_request_var( $post_type->query_var, false );
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: TRUE, FALSE and NULL must be uppercase; expected "FALSE" but found "false"
* :warning: Line exceeds 80 characters; contains 87 characters

<sub>hash: 860681149651c8f922819dcc1b5f2e7a</sub>